### DISCUSSION TITLE
cant docker-compose up -d

### DISCUSSION THREAD

**Discussion by isgm570** [photoview/photoview#1132] [Q&A and Troubleshooting] [answered]
when trying to enter it, this error shows up 

`error(s) decoding:`

`* error decoding 'volumes[3]': invalid spec: :/photos:ro: empty section between colons`

I downloaded docker-compose.yml from the linkj as stated in the guide, and i also didnt change anything.

**Comment by kkovaletp** [comment, marked as answer]
@Forendr, thanks for the question.
It looks like you forgot to download and set the .env file so that the variable in the compose file is resolved to an empty string.

Please make sure to carefully read and follow the docker setup guide in the readme.
